client/src/resources/map/MapView.js

```diff
@@ -50,20 +74,43 @@ class MapView extends React.Component {
             googleMapElement={
               <GoogleMap
                 ref="map"
-                scrollWheel='false'
-                defaultZoom={12}
+                scrollwheel={false}
```

Comment:
They're lucky! It was super frustrating to scroll zoom on the map. It's a moot issue since the map is the whole screen now.